src/odemis/driver/zeiss.py

```diff
@@ -985,13 +1069,48 @@ def _run_move(self):
                                      1)
                 traveled_dist = total_dist * traveled_ratio
                 self.pos = orig_pos + traveled_dist
+
+                # check here if the min/max boundary of the range is passed
+                # recalculate the difference to support under- and overflow
+                if self.pos[-2] < 0:
+                    # remainder should be subtracted from 360°
+                    bound_pos = 2 * math.pi - (0 - self.pos[-2])
+
+                elif self.pos[-2] > 2 * math.pi:
+                    # remainder should be added to 0°
+                    bound_pos = 0 + (self.pos[-2] - (2 * math.pi))
+
+                if bound_pos:
+                    self.pos[-2] = bound_pos
```

Comment:
I think you can just write it like that:
```suggestion
                # R is allowed to do any rotation, but always stays within 0 -> 360°
                self.pos[-2] %= 360
```